### DISCUSSION TITLE
How to join Beta testing

### DISCUSSION THREAD

**Discussion by arncor** [osmandapp/OsmAnd#15555] [Q&A]
Hi, 

I've been using Osmand+ for a few years and loving it. I previously used Fdroid version and recently bought the PlayStore version. I also try to contribute via Github and I'd love to test the Android beta but I can't find where I could download it.

In play store I don't find any option where I could subscribe to beta test.

Thanks

**Comment by pebogufi** [comment]
You can dnl here
https://download.osmand.net/latest-night-build/

**Reply by arncor** [reply]
Hey,
Thanks for the fast answer.
I tried to install:
- Osmand android Full 
- Osmand arm64 nightly
- Osmand arm nightly

But I got an "apk not installed" error.

Any ideas why?

**Comment by pebogufi** [comment]
You don't give information about your device, so it is not easy to understand what might be a problem.

**Reply by arncor** [reply]
Sure! Sorry about that.

It's an Ulefone Armor 6E running Android9. Kernel 4.4.146

**Comment by arncor** [comment]
quick update : I just managed to install the nightly build without any issue.

However, I can't download beta version because it seems I'm not part of the beta test program -> on Osmand official website, when trying to download a beta version of osmand, I'm redirected towards google login page and after login : "Your account isn't currently eligible for this app's testing program."

If I download it via your link it won't install (Due to the fact that it considers it as an update of the existing osmand+ I believe)

**Reply by jfburke3** [reply]
If you install the "night" build, you *can't* join the Play store beta program. You'd have to uninstall it, then reinstall from the Play store. 

As to why you couldn't join the beta program before installing the night build, I'm not sure. That's more of a question for the Google help forums.

**Reply by arncor** [reply]
Hi, thanks for the clarification.
In the meantime I finally managed to join the beta program in the play store. I don't know why I couldn't do that before but now it works.
